Software/Development/SemesterProject/App/Include/errorManager.h

```diff
@@ -25,6 +25,22 @@ void disableDrivers(void);
 void enableDrivers(void);
 void turnOnErrorLED(void);
 void turnOffErrorLED(void);
+
+void setCurrentAErrorStatus(void);

```

Comment:
what are this set..() guys for?

If this is after what I told you that we should be able to trigger errors from outside, I just meant to have a single interface called SWError or somehting.

Reply:
These are to set the errorStatus in the latching struct. I didnt really do anything to trigger an error